### PR TITLE
chore: adjust spacing to classNames

### PR DESCRIPTION
**Pull request by mrCherry97**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

**Definition of done**

- [ ] The PR's title starts with one of the following prefixes:
  - feat: A new feature
  - fix: A bug fix
  - docs: Documentation only changes
  - refactor: A code change that neither fixes a bug nor adds a feature
  - test: Adding tests
  - chore: Maintainance changes to the build process or auxiliary tools, libraries, workflows, etc.
- [ ] Related issues are linked. To link internal trackers, use the issue IDs like `backlog#4567`
- [ ] Explain clearly why you created the PR and what changes it introduces
- [ ] All necessary steps are delivered, for example, tests, documentation, merging

### CODE REVIEW

File: src/components/BusolaExtensions/BusolaExtensionDetails.js [hunk 1]
```diff
@@ -153,7 +152,7 @@ export function BusolaExtensionDetails({ name, namespace }) {
           <MessageStrip
             design="Information"
             hideCloseButton
-            style={spacing.sapUiSmallMarginBottom}
+            className="sap-margin-bottom-small sap-margin-top-small"
```
Comment (OliwiaGowor) [inline src/components/BusolaExtensions/BusolaExtensionDetails.js:155]:
```suggestion
            className="sap-margin-y-small"
```

File: src/components/Clusters/components/AuthForm.js [hunk 2]
```diff
@@ -129,7 +128,7 @@ export function AuthForm({
         <MessageStrip
           design="Warning"
           hideCloseButton
-          style={spacing.sapUiSmallMarginTopBottom}
+          className="sap-margin-top-small sap-margin-bottom-small"
```
Comment (OliwiaGowor) [inline src/components/Clusters/components/AuthForm.js:131]:
```suggestion
          className="sap-margin-y-small"
```

File: src/components/Clusters/components/ClusterPreview.js [hunk 3]
```diff
@@ -118,30 +86,17 @@ export function ClusterPreview({ kubeconfig, storage, setSelected, hasAuth }) {
...
       <div className="add-cluster__content-container">
-        <Title level="H5" style={spacing.sapUiMediumMarginBottom}>
+        <Title level="H5" className="sap-margin-bottom-medium">
           {t('clusters.wizard.review')}
         </Title>
         <Title
           level="H5"
-          className="cluster-preview__subtitle"
-          style={spacing.sapUiSmallMarginTopBottom}
+          className="cluster-preview__subtitle sap-margin-top-small sap-margin-bottom-small"
```
Comment (OliwiaGowor) [inline src/components/Clusters/components/ClusterPreview.js:94]:
```suggestion
          className="cluster-preview__subtitle sap-margin-y-small"
```

File: src/components/Clusters/components/ClusterPreview.js [hunk 4]
```diff
@@ -153,17 +108,10 @@ export function ClusterPreview({ kubeconfig, storage, setSelected, hasAuth }) {
         </div>
         <Title
           level="H5"
-          className="cluster-preview__subtitle"
-          style={spacing.sapUiSmallMarginTopBottom}
+          className="cluster-preview__subtitle sap-margin-top-small sap-margin-bottom-small"
```
Comment (OliwiaGowor) [inline src/components/Clusters/components/ClusterPreview.js:111]:
```suggestion
          className="cluster-preview__subtitle sap-margin-y-small"
```

File: src/components/Clusters/components/ClusterPreview.js [hunk 5]
```diff
@@ -177,25 +125,12 @@ export function ClusterPreview({ kubeconfig, storage, setSelected, hasAuth }) {
...
```
Comment (OliwiaGowor) [inline src/components/Clusters/components/ClusterPreview.js:128]:
```suggestion
          className="cluster-preview__subtitle sap-margin-y-small"
```

File: src/components/Clusters/components/ContextChooser/ContextChooser.js [hunk 6]
```diff
@@ -63,7 +59,7 @@ export function ContextChooser(params) {
           <MessageStrip
             design="Information"
             hideCloseButton
-            style={spacing.sapUiSmallMarginTopBottom}
+            className="sap-margin-top-small sap-margin-bottom-small"
```
Comment (OliwiaGowor) [inline src/components/Clusters/components/ContextChooser/ContextChooser.js:62]:
```suggestion
            className="sap-margin-y-small"
```

File: src/shared/components/ErrorBoundary/ErrorBoundary.js [hunk 7]
```diff
@@ -31,13 +29,10 @@ class ErrorBoundaryComponent extends React.Component {
...
               height: '70vh',
-              margin: spacing.sapUiMediumMargin.margin,
             }}
+            className="sap-margin-medium"
           >
-            <p
-              className="bsl-color--text"
-              style={spacing.sapUiSmallMarginTopBottom}
-            >
+            <p className="bsl-color--text sap-margin-top-small sap-margin-bottom-small">
```
Comment (OliwiaGowor) [inline src/shared/components/ErrorBoundary/ErrorBoundary.js:35]:
```suggestion
            <p className="bsl-color--text sap-margin-y-small">
```

**Review by OliwiaGowor — COMMENTED** [review]
Did we use all of the replaced style props only for margins and paddings? So it can be safely replaced with only classNames or removed. Files:

- SeparatorLine.js 
- DescriptionHint.tsx 
- ExternalLink.tsx
- UI5Panel.tsx

**Comment by mrCherry97** [comment]
SeparatorLint - my mistake, it's adjusted
DescriptionHint.tsx - `style` prop is nowhere in use so that it could be removed 
ExternalLink.tsx - `iconStyle` and `linkStyle` are adjusted everywhere to `linkClassName` and `iconClassName`
UI5Panel.tsx - `style` prop is nowhere in use so that it could be removed

**Review by OliwiaGowor — APPROVED** [review]

**Pull request merged**